### DISCUSSION TITLE
Part of the stitching is missing

### DISCUSSION THREAD

**Discussion by GJMY592** [OpenStitching/stitching#186] [Q&A] [answered]
Hello, I have merged 20+ photos taken on a single site, and the parameter is set --**no_crop**, and the merged result is obviously missing 【The missing locations accounted for about 10%, located in the upper left and upper right corners of the final image】 (logs are as follows).

StitchingWarning: Not all images are included in the final panorama.
                          If this is not intended, use the 'matches_graph_dot_file'
                          parameter to analyze your matches. You might want to
                          lower the 'confidence_threshold' or try another 'detector'
                          to include all your images.

In order to solve this problem, I have tried different detector and confidence_threshold, but there are still gaps. Please tell me whether there is any problem in the process of using stitcher

**Comment by lukasalexanderweber** [comment]
Maybe  -v will help you identify problematic images

**Reply by lukasalexanderweber** [reply]
"This will create a folder where all intermediate results are stored so that you can find out where there are problems with your images, if any"

**Comment by GJMY592** [comment, marked as answer]
When I set **verbose**, I see that all images have calculated feature points, but a few images are not matched, and among the previously missing parts, one appears in the matching phase, and the other images are not matched.

Then, I set confidence_threshold (orb-0.8) and an error occurred:
**stitching.stitching_error.StitchingError: Camera parameters adjusting failed.**

It's weird. I lowered the threshold and got the wrong one...

**Reply by lukasalexanderweber** [reply]
is your problem solved or not?

**Reply by GJMY592** [reply]
Not yet, I'm not sure I understand the difference between match_conf and confidence_threshold. I've tried various combinations (including adjuster-wave-warper, etc.) and the best result is to modify only --no_crop

**Reply by GJMY592** [reply]
The best result is still a missing area

**Reply by lukasalexanderweber** [reply]
https://github.com/OpenStitching/stitching/discussions/103 explains the difference between 'confidence_threshold' and 'match_conf'

**Reply by GJMY592** [reply]
Thank you for your patient answer, I understand, now I will continue to verify ：）

**Reply by rayanelahmadi** [reply]
Has the issue been figured out? I am running into a similar problem
